src/components/LeafletMap.js

```diff
@@ -40,20 +43,12 @@ const icon = L.divIcon({
 });
 
 class LeafletMap extends Component {
-  componentDidUpdate() {
-    const { geoObject, siteClickHandler } = this.props;
-    if (geoObject.length > 0) {
-      geoObject.map(site => {
-        return L.marker([site.geometry.coordinates[1], site.geometry.coordinates[0]], {
-          icon: icon
-        })
-          .addTo(this.map)
-          .on('click', () => {
-            siteClickHandler(site);
-          });
-      });
+  componentDidUpdate({ markersData }) {
```

Comment:
For readability, when destructuring the `prevProps` param of `componentDidUpdate`, you should incorporate the term "previous", "last", or "prior" to inform other developers that you're comparing against the last props before the component's data changed. 

For me, I'd do something like the following:
```js
componentDidUpdate({markersData: prevMarkersData}) {
  if (this.props.markersData !== prevMarkersData) {
  (...)
  }
}
```